package.json

```diff
@@ -1,11 +1,11 @@
 {
     "name": "@prk/eslint-config",
-    "version": "0.0.5",
+    "version": "0.0.6",
     "description": "Praekelt Consulting base ESLint config.",
     "repository": "git@github.com:praekelt/eslint-config.git",
     "main": "index.js",
     "license": "BSD-3-Clause",
-    "peerDependencies": {
+    "dependencies": {
         "eslint": ">= 4",
```

Comment:
so im thinking possibly eslint should be a peer dependancy. and the rest can be dependancies? since the point of this repo is to be a config.

Reply:
Good point. Will amend.